src/js/components/Peak/PeakDescription.tsx

```diff
@@ -24,13 +37,91 @@ const PeakDescription = ({ peak, showForm }: PeakDescriptionProps) => {
       <p>Województwo: {peak.region}</p>
       <p>Pasmo górskie: {peak.mountainRange}</p>
       <PeakStatistics peakId={peak.id} />
-      {showForm && (
-        <div className={styles.buttonBox}>
-          <SubmitButton
-            onClick={() => reachPeak(3000)}
-            text="Zaznacz jako zdobyty"
-            className={styles.reachPeakButton}
-          />
+      {true && (
```

Comment:
Ogólnie to wyciągnął bym to z tego przycisku i podpiał pod przycisk zdobądź na karcie szczytu. Nie jest nam potrzebny dodatkowy przycisk